public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -575,14 +578,102 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
 
                     isMeasurement = true;
 
+                    Float male_bin_min = null;
```

Comment:
We need to factor this function out some. Like here: https://github.com/all-of-us/workbench/blob/9132f08c3496685bad9e9cc002b31adaf1ab3c37/public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java#L524

inside that loop , have a function to do the custom stuff for each analysis_id ..
What Do you think?

Reply:
Yeah, seperated the analysis processing into different functions.